src/ROUTES.ts

```diff
@@ -306,6 +307,82 @@ const ROUTES = {
     MONEY_REQUEST_MANUAL_TAB: ':iouType/new/:reportID?/manual',
     MONEY_REQUEST_SCAN_TAB: ':iouType/new/:reportID?/scan',
 
+    MONEY_REQUEST_CREATE: {
+        route: 'create/:iouType/start/:transactionID/:reportID',
+        getRoute: (iouType: ValueOf<typeof CONST.IOU.TYPE>, transactionID: string, reportID: string) => `create/${iouType}/start/${transactionID}/${reportID}` as const,
+    },
+    MONEY_REQUEST_STEP_CONFIRMATION: {
+        route: 'create/:iouType/confirmation/:transactionID/:reportID/',
+        getRoute: (iouType: ValueOf<typeof CONST.IOU.TYPE>, transactionID: string, reportID: string) => `create/${iouType}/confirmation/${transactionID}/${reportID}/` as const,
+    },
+    MONEY_REQUEST_STEP_AMOUNT: {
+        route: 'create/:iouType/amount/:transactionID/:reportID/',
```

Comment:
Coming from https://github.com/Expensify/App/issues/33205#issuecomment-1860193100

The routes added here have a trailing `/`. It lead to inconsistenecy because our routes don't have trailing slashes.

There was also a  bug that broke "go back"  because our code didn't handle trailing slashes.